compositor_pipeline/src/audio_mixer/prepare_inputs.rs

```diff
@@ -71,8 +76,8 @@ fn frame_input_samples(
     let time_to_sample_count = |duration: Duration| {
         let sample_count = duration.as_secs_f64() * sample_rate as f64;
         // If value is close to the integer then round it, otherwise fallback to standard
-        // integer division behavior.
-        if (sample_count - sample_count.round()).abs() < max_error.as_secs_f64() {
+        // integer division behavior. Close is defined as 1% of a sample (the same as max_error).
```

Comment:
[NIT]
```suggestion
        // Integer division behavior. Close is defined as 1% of a sample (the same as max_error).
```

Reply:
This is not the start of the sentence, check line above

Reply:
Ok, nevermid, my bad :)